app/components/object-inspector/dependent-keys.hbs

```diff
@@ -1,14 +1,17 @@
 <li class="mixin__property-dependency-list relative">
   {{svg-jar "dependent-key-connection" width="20px" height="10px"}}
   <ul class="m-0 p-0 list-none">
-    {{#each @keys as |depKey|}}
-      <li class="mixin__property-dependency-item relative text-base12 text-sm">
-      {{#if (match depKey "•")}}
+    {{#each @keys as |dep|}}
+      <li
+              class="mixin__property-dependency-item relative text-base12 text-sm {{if dep.changed 'text-spec01'}}"
```

Comment:
It looks like there is extra spacing here, can we remove that?

Reply:
fixed